wallet/src/de/schildbach/wallet/ui/dashpay/CreateIdentityService.kt

```diff
@@ -279,14 +279,14 @@ class CreateIdentityService : LifecycleService() {
         }
 
         if (blockchainIdentityData.creationState <= CreationState.PREORDER_REGISTERING) {
-            platformRepo.updateCreationState(blockchainIdentityData, CreationState.PREORDER_REGISTERING)
             //
             // Step 4: Preorder the username
             if (!blockchainIdentity.getUsernames().contains(blockchainIdentityData.username!!)) {
                 blockchainIdentity.addUsername(blockchainIdentityData.username!!)
             }
             platformRepo.preorderNameAsync(blockchainIdentity, encryptionKey)
             platformRepo.updateBlockchainIdentityData(blockchainIdentityData, blockchainIdentity)
+            platformRepo.updateCreationState(blockchainIdentityData, CreationState.PREORDER_REGISTERING)
```

Comment:
Setting the creation state should be after the operation starts/ends, not before.

Reply:
Actually, it doesn't matter, since there is a <= above